source/nodejs/adaptivecards-site/themes/adaptivecards/layout/archive.ejs

```diff
@@ -51,6 +119,10 @@ layout: layout
 	<% } %>
 	<% }); %>
 
+	<% if(page.posts.length % 3 !== 0) { /* if the posts were not divisible by 3, we need to close the last row */ %>
+	</div>
+	<% } %>
+
 	<!-- Pager -->
```

Comment:
QQ @paulcam206 What is the Pager for? Is it added for when we eventually have more blogs?

Reply:
looks like an [option](https://hexo.io/docs/configuration#Pagination) we're not currently using:

https://github.com/microsoft/AdaptiveCards/blob/09b0c8adaa2da5d2f5d8879733f6eabd1d85608b/source/nodejs/adaptivecards-site/_config.yml#L70-L73

Reply:
Ah interesting. Should we remove it or just keep it around in case we want it one day?